assets/javascripts/discourse/initializers/tdc-user-model.js.es6

```diff
@@ -0,0 +1,52 @@
+// Extend discourse/app/assets/javascripts/discourse/models/user.js.es6
+// The vanilla createAccount method doesn't pass custom fields
+import { ajax } from 'discourse/lib/ajax';
+
+import { userPath } from 'discourse/lib/url';
+import { withPluginApi } from 'discourse/lib/plugin-api';
+import InputValidation from 'discourse/models/input-validation';
+import createAccountController from 'discourse/controllers/create-account';
+import User from 'discourse/models/user'
+
+import DebtAmountValidation from "../mixins/debt-amount-validation";
+
+function initializeTdcUser(api) {
+  // use this when/if my PR is approved
+  //api.modifyClass('model:user', {
```

Comment:
Do we still need to do this?

Reply:
I'm not sure I understand what you're asking. Can you rephrase or expand?

Reply:
The comment on 14 says to use the commented out part of 15 when the PR is merged. At least that's what I understood it to mean, maybe it's something else though!  Just wondering what you meant/if we need to do anything with this before merging or if it's just left overs.